web/src/features/llm-api-key/server/router.ts

```diff
@@ -109,22 +107,28 @@ export const llmApiKeyRouter = createTRPCRouter({
         scope: "llmApiKeys:read",
       });
 
-      const apiKeys = z.array(LlmApiKey).parse(
-        await ctx.prisma.llmApiKeys.findMany({
-          // we must not return the secret key via the API, hence not selected
-          select: {
-            id: true,
-            createdAt: true,
-            updatedAt: true,
-            provider: true,
-            displaySecretKey: true,
-            projectId: true,
-          },
-          where: {
-            projectId: input.projectId,
-          },
-        }),
-      );
+      const apiKeys = z
+        .array(LLMApiKeySchema.extend({ secretKey: z.undefined() }))
```

Comment:
cant we remove `.extend({ secretKey: z.undefined() })` as we do not select this?